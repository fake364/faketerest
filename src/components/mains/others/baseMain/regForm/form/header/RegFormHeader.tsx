import React from 'react';
import Image from 'next/image';
import useTranslation from 'next-translate/useTranslation';

type Props = {};

const RegFormHeader: React.FC<Props> = () => {
  const { t } = useTranslation('main-page');

  return (
    <>
      <div className="my-[16px]">
        <Image src={'/logo.png'} width={40} height={40} />
      </div>
      <div className="text-[36px] text-[#333333] font-semibold">
        {t('regForm.form.title')}
      </div>
      <div className="text-[16px] text-[#333333] mt-[8px]">
        {t('regForm.form.subtitle')}
      </div>
    </>
  );
};

export default RegFormHeader;
