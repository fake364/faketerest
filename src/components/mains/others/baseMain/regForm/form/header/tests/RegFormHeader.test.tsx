import { shallow } from 'enzyme';
import RegFormHeader from '../RegFormHeader';
import React from 'react';

describe('RegFormHeader', () => {
  it('should render reg form header with logo', () => {
    const wrapper = shallow(<RegFormHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
