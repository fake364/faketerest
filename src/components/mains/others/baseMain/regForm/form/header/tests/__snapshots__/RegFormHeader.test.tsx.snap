// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegFormHeader should render reg form header with logo 1`] = `
<Fragment>
  <div
    className="my-[16px]"
  >
    <Image
      height={40}
      src="/logo.png"
      width={40}
    />
  </div>
  <div
    className="text-[36px] text-[#333333] font-semibold"
  >
    regForm.form.title
  </div>
  <div
    className="text-[16px] text-[#333333] mt-[8px]"
  >
    regForm.form.subtitle
  </div>
</Fragment>
`;
